jsapp/js/components/accountSettings.es6

```diff
@@ -144,6 +152,9 @@ export var AccountSettings = React.createClass({
       background: `#${stringToColor(accountName)}`
     };
 
+    let bHeight = $('textarea#bio').prop('scrollHeight');
```

Comment:
Note: In general, this kind of call to the DOM should be moved out of the render function